### PR TITLE
Change images

### LINKED ISSUE #381: Functionality for update users
## Description
Currently, there is no ability to update user information. Requires navigation guard. This needs to be addressed and fixed.

## Acceptance criteria
- [ ] As an admin you should be able to update user information (the user image) 

## Solution proposal:
In users/:id, add a simple form for updating the image.

### PR DESCRIPTION
**Pull request by te4-alexander-nylundgomes**
## ⚙️ What issue did you implement or fix?
Fixes #381 

## 📜 Summary
- Added functionality to change images if user is logged in as admin

## 📝 Type of change

Please delete options that are not relevant.

- [x] ✨ New feature or enhancement (non-breaking change which adds functionality)

## ✅ Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## 💩 How can this implementation be improved?
- Could use testing

### CODE REVIEW

**Comment by te4-alexander-kjellberg1** [comment]
Why did "Integration / api-tests (push)" fails? is it possible to fix?

File: frontend/src/views/ProfileView.vue [hunk 1]
```diff
@@ -10,13 +10,66 @@
...
+                />
+              </v-row>
+            </template>
+          </v-img>
         </v-card>
       </v-col>
     </v-row>
+
+    <v-card
+      v-if="RoleChecker.isAdmin()"
```
Comment (te4-alexander-kjellberg1) [inline frontend/src/views/ProfileView.vue:37]: What does isAdmin do?

Comment (te4-alexander-nylundgomes) [reply frontend/src/views/ProfileView.vue:37]: Checks if current user is admin. If not, there should be no possibility of changing the persons image.

**Comment by te4-alexander-nylundgomes** [comment]
> Why did "Integration / api-tests (push)" fails? is it possible to fix?

Tests fail because the test is not functioning properly.

**Review by TE4-David-Sundqvist — APPROVED** [review]
All looks good 👍

**Review by te4-alexander-kjellberg1 — APPROVED** [review]
oki, lookin good

**Pull request merged**